HCAL DQM Layouts

vkhristenko opened the pull request:
Synching newest layouts for online DQM with CMSSW updates deployed after Readiness Review in April 2016.
This PR will include Layouts for Offline DQM as well

VK

vanbesien commented:
Ciao,

- Verbosity is still on 10
- Can you explain shift layout? Is now numbered 0 1 3 5 28 30, while the instructions will only ask to look at the summary.
- You're modifying the L1T workspace

vkhristenko commented:
- fixed the verbosity
- All the other ones define the status setting, HCAL is not performing optimally - if it's neccessary in short term instructions to watch out for something else in addition it will one of the quantities already in the shift layout. But, yes, instructions will be simple to look at Run Summary
- L1T? ah, I see.... fixed should be like it was before my updates now

vkhristenko commented:
closing and it's superseeded by 318
